### PR TITLE
Added install directives for the binary library

### PR DESCRIPTION
**Pull request by lounick**
Guidance provides a binary library. It has to be installed in the
develspace in order for other packages to find it.

@meyerj PTAL

### CODE REVIEW

**Comment by meyerj** [comment]
We did not discuss that yesterday, but please also add the necessary install rules:
http://wiki.ros.org/catkin/CMakeLists.txt#Optional_Step:_Specifying_Installable_Targets

**Comment by lounick** [comment]
~~Should be done now.~~ Nope I get a build error. How can I specify that the only thing I want installed is the library?

**Comment by lounick** [comment]
Ok now it just installs the library file and builds fine. I think that since we don't want anything else from this package than the lib and the header it is fine to be like that.

File: CMakeLists.txt [hunk 1]
```diff
@@ -35,9 +37,22 @@ endif ()
 if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "armv7l" )
   message("-- " ${CMAKE_SYSTEM_PROCESSOR} " detected")
   link_directories(lib/XU3)
+  set(DJI_guidance_LIBRARY_DIR lib/XU3)
 else ()
 endif ()
 
+catkin_package(
```
Comment (meyerj) [inline CMakeLists.txt:41]: Please remove the original `catkin_package()` call above.

Comment (lounick) [reply CMakeLists.txt:41]: Done

**Pull request merged**